R/discovery.R

```diff
@@ -77,7 +78,9 @@ gar_create_package <- function(api_json,
   
   if(github){
     ## if fail, use git add
-    devtools::use_github(pkg = package_dir, protocol = "https")
+    usethis::with_project(package_dir,
```

Comment:
@jimhester may I ask `with_project()`'s behaviour, will it use the same folder as the issuing RStudio project?

Reply:
`with_project()` will temporarily set the usethis project to the first argument, then run the code in the second.

Reply:
Thanks Jim